readinessProbe/readiness/suite_test.go

```diff
@@ -89,11 +97,31 @@ var _ = BeforeSuite(func(done Done) {
 
 	go RunServer(k8sClient, types.NamespacedName{Name: ManagedOCSName, Namespace: TestNamespace}, ctrl.Log.WithName("readiness"))
 
+	ctx := context.Background()
+
+	managedOCS := &v1.ManagedOCS{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      ManagedOCSName,
+			Namespace: TestNamespace,
+		},
+	}
+	Expect(k8sClient.Create(ctx, managedOCS)).Should(Succeed())
+	utils.WaitForResource(k8sClient, ctx, managedOCS, timeout, interval)
```

Comment:
You just created the`managedOCS` resource and validated the creation., why do we need the WaitForResource on the same resource?

Reply:
That is a good point. This additional check was not needed.